apps/staking/app/register/[nodeId]/solo/SubmitSoloTab.tsx

```diff
@@ -80,8 +98,8 @@ export function SubmitSoloTab() {
     amount: SESSION_NODE_FULL_STAKE_AMOUNT,
   });
 
-  const addBlsFeeParams = useMemo(() => {
-    return {
+  const { addBlsFeeParams, addBlsVestingFeeParams } = useMemo(() => {
+    const addBlsFeeParams = {
```

Comment:
I was able to do `const addBlsFeeParams: UseAddBlsPubKeyParams` and remove the `satisfies` and I got no errors and everything built correctly?

Reply:
satisfies is better here tho, we dont want to assign a type (use `:`) we want the object to satisfy being the type we need it to be